cryptographic_estimators/MQEstimator/MQAlgorithms/f5.py

```diff
@@ -135,13 +122,13 @@ def _time_complexity_regular_system(self):
         """
         Return the time complexity for regular system
 
-        EXAMPLES::
+        TEST::
 
             sage: from cryptographic_estimators.MQEstimator.MQAlgorithms.f5 import F5
             sage: from cryptographic_estimators.MQEstimator.mq_problem import MQProblem
-            sage: E = F5(MQProblem(n=10, m=5, q=3))
-            sage: E._time_complexity_regular_system()
-            12.643856189774725
+            sage: E = F5(MQProblem(n=10, m=5, q=31), bit_complexities=False)
+            sage: E.time_complexity()
```

Comment:
This is not testing the actual function. If this function is testable via the time_complexity we do not need an extra test here. Then a test for the time_complexity will suffice